routes/web.php

```diff
@@ -10,7 +10,11 @@
 | contains the "web" middleware group. Now create something great!
 |
 */
-
-Route::get('/', function () {
-    return view('welcome');
-});
+Route::get('/', 'HomeController@index')->name("home.index");
+Route::get('/forum', 'HomeController@forum')->name("home.forum");
+Route::get('/post/show/{id}', 'PostController@show')->name("post.show");
+Route::get('/post/create', 'PostController@create')->name("post.create");
+Route::post('/post/save', 'PostController@save')->name("post.save");
+Route::get('/post/list', 'PostController@list')->name("post.list");
+Route::delete('/post/destroy/{id}', 'PostController@destroy')->name("post.destroy");
```

Comment:
Use:

GET /post/{id}
GET /post/new
POST /post/save
GET /post/index
DESTROY /post/{id}

instead :)

Reply:
Destroy instead of delete?

Reply:
My bad. Use DELETE

Reply:
Done

Reply:
Why /post/{id} instead of /post/show/{id} ? I think last one is more descriptive.